packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/athena/start-query-execution.ts

```diff
@@ -209,6 +216,7 @@ export class AthenaStartQueryExecution extends sfn.TaskStateBase {
           OutputLocation: this.props.resultConfiguration?.outputLocation ? `s3://${this.props.resultConfiguration.outputLocation.bucketName}/${this.props.resultConfiguration.outputLocation.objectKey}/` : undefined,
         },
         WorkGroup: this.props?.workGroup,
+        ExecutionParameters: this.props.executionParameters,
```

Comment:
A `private validateExecutionParameters(executionParameters?: string[])` method should be created to apply validation according to [this](https://docs.aws.amazon.com/athena/latest/APIReference/API_StartQueryExecution.html#athena-StartQueryExecution-request-ExecutionParameters) specs.
The method should be invoked in the constructor.